homeassistant/components/homeassistant_yellow/strings.json

```diff
@@ -99,5 +99,11 @@
       "install_addon": "[%key:component::homeassistant_hardware::silabs_multiprotocol_hardware::options::progress::install_addon%]",
       "start_addon": "[%key:component::homeassistant_hardware::silabs_multiprotocol_hardware::options::progress::start_addon%]"
     }
+  },
+  "issues": {
+    "cm4_unseated": {
+      "title": "Raspberry Pi Compute Module is not firmly installed",
+      "description": "Your Raspberry Pi Compute Module does not appear to be plugged in all the way and some pins are disconnected. Carefully disassemble the Yellow and securely click the Compute Module into place."
```

Comment:
I think we should link also here https://yellow.home-assistant.io/power-supply/#installing-the-cm4, if possible and make it more clear we want the user to first remove the heatsink and CM4 and then reinstalling the CM4 and heatsink.

Maybe it's worth the effort of creating a new page which is about removing and then reseating the CM4?

Reply:
cc @c0ffeeca7

Reply:
There is a section describing how to remove the CM4
<https://yellow.home-assistant.io/guides/remove-cm4/>

Reply:
oh, I see. you mean a one pager you can link to. Will do.

Reply:
<https://github.com/home-assistant/yellow.home-assistant.io/pull/247>